README.md

```diff
@@ -287,11 +285,10 @@ app.get("/sign_up", passport.authenticate(WebAppStrategy.STRATEGY_NAME, {
 }));
 ```
 Note:
-1. If your Cloud directory setting "Allow users to sign-in without email verification" set to "No", the process will end without retrieving App ID access and id tokens.
-2. Make sure to set "Allow users to sign up and reset their password" to ON, in Cloud Directory settings that are in AppID dashboard.
```

Comment:
why the second note was omitted ?

Reply:
Actually - I don't know. I think it was an accident. It can come back. Sorry!

Reply:
can u fix the pull request, I can edit it because there isn't a supported branch

Reply:
done!